app/ai/voice/agents/breeze_buddy/agent/__init__.py

```
@@ -899,6 +900,32 @@ async def run(self, runner_args: Optional[RunnerArguments] = None) -> None:
+                    tts, tts_map, agent_templates_map = (
+                        await build_orchestrator_tts_switcher(
+                            self.configurations,
+                            call_payload=lead_payload,
+                        )
+                    )
+                    logger.info(
+                        f"[Orchestrator] Orchestrator TTS switcher built with "
+                        f"{len(tts_map)} service(s); "
+                        f"{len(agent_templates_map)} agent template(s) pre-rendered"
+                    )
+                except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)